lib/pubsub/subscription.js

```diff
@@ -280,16 +277,6 @@ function Subscription(pubsub, options) {
   this.messageListeners = 0;
   this.paused = false;
 
-  if (is.number(options.timeout)) {
-    this.timeout = options.timeout;
-  } else {
-    // The default timeout used in gcloud-node is 60s, but a pull request times
-    // out around 90 seconds. Allow an extra couple of seconds to give the API a
-    // chance to respond on its own before terminating the connection.
-    var PUBSUB_API_TIMEOUT = 90000;
-    this.timeout = PUBSUB_API_TIMEOUT + 2000;
```

Comment:
Not really sure if it's still relevant, but after a little bit of searching it would seem we could sort of simulate a timeout by checking if we've received any data after a certain period of time and then canceling the connection.

I could be wrong but I think it might end up looking something like..

```js
var timeout;
var call = service[protoOpts.method](reqOpts, function() {
  clearTimeout(timeout);
});
timeout = setTimeout(call.cancel.bind(call), 90000);
```